developer_manual/app/code_signing.rst

```diff
@@ -139,6 +139,12 @@ In case you lose your certificate please submit a new CSR as described above and
 mention that you have lost the previous one. ownCloud will revoke the old 
 certificate.
 
+If you're maintaining an app together with multiple people it is recommended to
+designate a release manager that is responsible for the signing process as well
```

Comment:
Sounds good, just two small wording fixes: »you’re maintaining« -> »you maintain«. And in »a release manager that is responsible« cut the »that is«.